Dockerfile

```diff
@@ -10,9 +10,9 @@
 #
 # http://github.com/jhowardmsft/busybox
 
-FROM microsoft/windowsservercore
+FROM mcr.microsoft.com/windows/servercore:ltsc2022
 RUN mkdir C:\tmp && mkdir C:\bin
-ADD http://frippery.org/files/busybox/busybox.exe /bin/
+RUN curl -LO http://frippery.org/files/busybox/busybox.exe
```

Comment:
Can you either put the file in `C:\bin` so next line with `setx` is still relevant or just remove it.

Reply:
Ah didn't catch the mklink, looks good then

Reply:
Is there a reason to switch this to use `curl` instead of `ADD`? `ADD` may also be able to check if the cache can be used.


If we switch to `RUN`, perhaps it would be good to combine the `RUN` steps into a single one (ISTR layers come with a larger overhead on windows images than on Linux, so reducing layers may be relevant?


Also, if we _do_ decide to switch, it'd be good to use the TLS URL, and the `-f` (fail fast), `-s` (silent), `-S` (show error), and `-L` (follow redirects) options;

```suggestion
RUN curl -fsSLO https://frippery.org/files/busybox/busybox.exe
```


Maybe we should add checksum validation as well (provided on that website); https://frippery.org/busybox/index.html, but that could be a follow-up

Reply:
For some reason, the server was returning 403 for `ADD`. Let me incorporate the proposed changes.

Reply:
Interesting 🤔 no idea why that would be, unless they blocked the user-agent? I wondered if there was a redirect in place, but that also doesn't seem to be (at least from the headers I see);


```bash
curl -I http://frippery.org/files/busybox/busybox.exe
HTTP/1.1 200 OK
Date: Thu, 04 Apr 2024 11:25:31 GMT
Server: Apache/2.4.38 (Debian)
Vary: User-Agent
Last-Modified: Wed, 21 Feb 2024 09:42:14 GMT
ETag: "9620e-611e12443c980"
Accept-Ranges: bytes
Content-Length: 614926
Content-Type: application/x-msdos-program
```